lua/r/config.lua

```diff
@@ -286,17 +286,17 @@ local hooks = require("r.hooks")
 ---@field rmd_environment? string
 ---
 ---Controls if and how backticks are replaced with code chunk/inline code
----delimiters when writing R Markdown and Quarto files. Do `:help rmdchunk`
----for more information.
----@field rmdchunk? string
+---delimiters when writing R Markdown and Quarto files.
+---Do `:help rmd_chunk_keymap` for more information.
```

Comment:
I am getting this message:

```
E149: Sorry, no help for rmd_chunk_keymap
```

Reply:
Maybe lazy.nvim didn't call `:helptags`. It should work if you do it manually:

```
:helptags /path/to/R.nvim/doc
```

Reply:
It is working now. Not sure why it was not when I used the PR checkout.